jobs/resource_job.go

```diff
@@ -113,24 +113,20 @@ type DbtTask struct {
 
 // EmailNotifications contains the information for email notifications after job or task run start or completion
 type EmailNotifications struct {
-	OnStart               []string `json:"on_start,omitempty"`
-	OnSuccess             []string `json:"on_success,omitempty"`
-	OnFailure             []string `json:"on_failure,omitempty"`
-	NoAlertForSkippedRuns bool     `json:"no_alert_for_skipped_runs,omitempty"`
-	AlertOnLastAttempt    bool     `json:"alert_on_last_attempt,omitempty"`
+	OnStart                    []string `json:"on_start,omitempty"`
+	OnSuccess                  []string `json:"on_success,omitempty"`
+	OnFailure                  []string `json:"on_failure,omitempty"`
+	OnDurationWarningThreshold []string `json:"on_duration_warning_threshold_exceeded,omitempty"`
```

Comment:
nit: Let's name this exactly like the underlying field (so that migration to Go SDK is a smaller change)